### PR TITLE
Bluetooth: controller: Fixes related to conformance testing

### PR DESCRIPTION
**Pull request by myrdyr**
Fixes TP/CON/MAS/BV-09-C [Master Accepting Termination] in LL.TS.5.0.0.

Fixes TP/CON/ADV/BV-03-C [Master Missing Slave Packets] in LL.TS.5.0.0.

### CODE REVIEW

**Review by cvinayak — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

File: subsys/bluetooth/controller/ll_sw/ctrl_internal.h [hunk 1]
```diff
@@ -90,18 +90,25 @@ struct connection {
...
-		u32_t window_widening_periodic_us;
-		u32_t window_widening_max_us;
-		u32_t window_widening_prepare_us;
-		u32_t window_widening_event_us;
-		u32_t window_size_prepare_us;
-		u32_t window_size_event_us;
-		u32_t force;
-		u32_t ticks_to_offset;
-	} slave;
+	union {
```
Comment (cvinayak) [inline subsys/bluetooth/controller/ll_sw/ctrl_internal.h]: It had to be dont in separate commit and it is clear the way it is commit now (with the use of anonymous union here).

**Review by cvinayak — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]
